Tests/Data/Elliptic/square_100x100_ComponentTransport/square_1e4_heterogeneity.prj

```diff
@@ -114,14 +123,9 @@
             <value>0</value>
         </parameter>
         <parameter>
-            <name>beta_l</name>
-            <type>Constant</type>
-            <value>1</value>
-        </parameter>
-        <parameter>
-            <name>beta_t</name>
+            <name>beta</name>
```

Comment:
I would keep the previous version since it is in agreement with the documentation.

Reply:
Fixed.